### PR TITLE
K8SPXC-1476: Disable PVC resize by default

### PR DESCRIPTION
**Pull request by egegunes**
[K8SPXC-1476](https://jira.percona.com/browse/K8SPXC-1476) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
*Short explanation of the problem.*

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?
- [ ] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are all needed new/changed options added to the [Helm Chart](https://github.com/percona/percona-helm-charts)?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PXC version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

[K8SPXC-1476]: https://perconadev.atlassian.net/browse/K8SPXC-1476?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/cw-bundle.yaml [hunk 1]
```diff
@@ -11318,7 +11320,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: percona/percona-xtradb-cluster-operator:1.15.0
+        image: perconalab/percona-xtradb-cluster-operator:main
         imagePullPolicy: Always
```
Comment (nmarukovich) [inline deploy/cw-bundle.yaml:11324]: why do we have main here?

Comment (nmarukovich) [reply deploy/cw-bundle.yaml:11324]: Should we update initContainer too? 
not critical at all but can be confusing

Comment (nmarukovich) [reply deploy/cw-bundle.yaml:11324]: We need to change percona/percona-xtradb-cluster-operator:1.15.0-logcollector-fluentbit3.1.4 and backup images?

Comment (egegunes) [reply deploy/cw-bundle.yaml:11324]: it was a mistake, fixed main. other images will be updated by @eleo007

**Review by hors — APPROVED** [review]

**Comment by benlangfeld** [comment]
> Problem:
> Short explanation of the problem.

🤔

**Comment by egegunes** [comment]
> > Problem:
> > Short explanation of the problem.
> 
> 🤔

@benlangfeld these PRs made in a rush to provide hotfix releases. Problem was some users reported problems that operator tries to resize PVCs after an upgrade on platforms which doesn't have a storage class that supports resizing. That's why we decided to disable this feature by default and you can enable it by using `spec.enableVolumeExpansion`.

**Pull request merged**